### DISCUSSION TITLE
acl [Jun 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2554] [Discord Threads]
> 🙋‍♂️ **JB_Denton**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### row acl is not possible at the moment  
#  
> 🙋‍♂️ **JB_Denton**  
#### Would you see it being included in the future or even possible with the current design?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/990170979072368650)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
